static/admin.html

```diff
@@ -1,54 +1,46 @@
-          'params': {'key': order.id}
-        }).success(function(data) {
-          $scope.refresh();
-        });
-      };
-      $scope.archive = function(order) {
-        $http({
-          'method': 'POST',
-          'url': '/api/archive_drink',
-          'params': {'key': order.id}
-        }).success(function(data) {
-          $scope.refresh();
-        });
-      };
-      $scope.refresh = function () {
-        $http.get('/api/drink_queue').success(function(data) {
-          console.log("refresh", data);
-          $scope.queue = data;
-        });
-      }
-
-      $scope.refresh();
-    }]);
-  </script>
-<body ng-controller="n8AdminCtrl">
-  <button ng-click="refresh()">Refresh</button>
-  <ul>
-    <li ng-repeat="order in queue">
-      <h4>{{order.drink_name}} for {{order.user_name}}</h4>
-      <span>{{order.total_oz}} oz</span>
-      <span ng-repeat="i in order.ingredients">
-        {{i.name}}
-      </span>
-      <button ng-click="approve(order)" ng-disabled="order.approved">Approve</button>
-      <button ng-click="archive(order)">Archive</button>
-    </li>
-  </ul>
+<head>
+  <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/angular_material/0.11.4/angular-material.min.css">
+  <link rel="stylesheet" href="app.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=RobotoDraft:300,400,500,700,400italic">
+  <link rel="stylesheet" href="app.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.7/angular.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.7/angular-animate.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.7/angular-cookies.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.7/angular-route.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.7/angular-aria.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angular_material/0.11.4/angular-material.min.js"></script>
+  <script src="compiled.js"></script>
+</head>
+<body ng-controller="n8AdminCtrl as ctrl">
+  <div layout="row" layout-align="center center">
+    <md-button class="md-fab" ng-click="ctrl.refresh()">
+      <md-icon md-svg-src="icons/ic_refresh_white_24px.svg"></md-icon>
+    </md-button>
+  </div>
+  <md-list class="admin-order-list">
+    <div>
+      <md-list-item class="md-3-line" ng-repeat="order in ctrl.queue">
+        <div class="md-avatar"
+             ng-class="{'admin-queue-position': order.position}">
+          <span ng-if="order.position">{{order.position}}</span>
+        </div>
+        <div class="md-list-item-text">
+          <h3>{{order.drink_name}} for {{order.user_name}}</h3>
+          <p>{{order.total_oz}} oz | {{ctrl.ingredientsCsv(order)}}</p>
+          <div>
+            <md-button class="md-primary"
```

Comment:
I think having a single combined button mitigates the chance that the buttons will get unaligned. But that's just the paranoid HTML developer in me.